### ISSUE TITLE
[bug] Space-filling log file from forward app

### ISSUE THREAD

**Issue by vovan-ivanoff** [guppyexpress/Pimax-Eye-Tracking-Fix#4]
If you dont use ASeeVRShutdown app to stop eye tracking sevice and just from taskbar close main ASeeVR app, forwarding application will continue running trying to connect to aseevr. Every time connection fails (for obvious reasons), it adds to logfile  line like this `[2024-10-01 23:52:00.097]: 2 socket err: 10054 -1 596` it does that multiple times a millisecond, so log file of forward app becomes ENOURMOUS. mine was 106 Gigs and it was produced in just shy of 3 hours.

In conclusion:
always quit both ASeeVR and forwarder or use  ASeeVRShutdown. Otherwise useless logs will be created as long as C: has some space left...
